api/libproject/devstart.rb

```diff
@@ -1060,6 +1060,38 @@ def backfill_gsuite_user_data(cmd_name, *args)
   :fn => ->(*args) {backfill_gsuite_user_data("backfill-gsuite-user-data", *args)}
 })
 
+def fetch_firecloud_user_profile(cmd_name, *args)
+  common = Common.new
+  ensure_docker cmd_name, args
+
+  op = WbOptionsParser.new(cmd_name, args)
+  op.opts.project = TEST_PROJECT
+
+  op.add_typed_option(
+      "--username=[username]",
+      String,
+      ->(opts, v) { opts.username = v},
+      "The AoU username to fetch FireCloud data for (e.g. 'gjordan'")
```

Comment:
What happens if you pass in `gjordan@fake-research-aou`?

Reply:
It probably errors out. I didn't think edge cases were too important for a demo command-line tool.

Reply:
Fair. I just know we often use the full email address as username in the DB, so wanted to make sure we handled that? (If I query the db, copy the username field, and run the tool, it will fail. Similarly if we want to run this impersonation powered by the user object in the DB, I am a little worried about failure there.)